include/etcpal/cpp/thread.h

```diff
@@ -173,6 +173,7 @@ class Thread
   Thread& SetName(const char* name) noexcept;
   Thread& SetName(const std::string& name) noexcept;
   Thread& SetPlatformData(void* platform_data) noexcept;
+  Thread& SetShutdownTimeout(unsigned ms_timeout) noexcept;
```

Comment:
Parameter type should be `unsigned int` for consistency with the other functions above.